projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -326,6 +400,24 @@ public function jetpack_user_table_row_actions( $actions, $user_object ) {
 					esc_html__( 'Revoke invite', 'jetpack' )
 				);
 			}
+			if ( current_user_can( 'edit_users' ) && $has_pending_invite ) {
+				$nonce                        = wp_create_nonce( 'jetpack-sso-resend-user-invite' );
+				$invite_nonce                 = wp_create_nonce( 'jetpack-sso-invite-user' );
```

Comment:
do we need this second nonce?

Reply:
@heavyweight feel free to test if it is needed, and if not remove it :)